tests/smoke.rs

```diff
@@ -445,6 +458,32 @@ fn dependencies() {
     run!()
 }
 
+/// Test building a dependency specified as `dependencies` in Xargo.toml
```

Comment:
I don't understand this comment. I thought this is about `--manifest-path`?

Reply:
fixed